PlexTrac/plextrac-manager-util
# QUICK-722 Remove default envs vars from couchbase-migration container

# This PR is in draft status until we confirm that using empty strings for these env vars will give us the intended behavior in the app code

## What
Changes the default values to be _**empty strings**_ for the `PG_TENANTS_WRITE_MODE` and `PG_TENANTS_READ_MODE` env vars in the `couchbase-migration` container.  ~It's now possible for these env vars to be "undefined" in the docker environment.~

## Why
1. The default values for these env vars is being managed in the application code because there's some logic that needs to be applied before choosing a default.
2. The outage described by https://plextrac.atlassian.net/browse/QUICK-721 resulted from the wrong default being applied for these env vars. Removing the defaults from the docker config will help to consolidate all the logic into one spot in the application code.

## Testing
1. Make sure that you don't have either of these env vars defined in your `.env` file.
2. Create a PlexTrac instance using the PlexTrac manager utility
3. Verify that the values being used for these env vars match the defaults set in `libs/dbDualWrite/src/dualWriteConfig.ts`:
    a. You should see logs indicating that the "Tenants ETL" was SKIPPED (because the write_mode defaulted to couchbase_only)
    b. Change the default WRITE_MODE (in the `dualWriteConfig.ts`) to `dual_write` and restart the 'couchbase-migration' container.  You should see that the "Tenants ETL" runs this time because the write_mode defaulted to "dual_write".
